### ISSUE TITLE
Training on dennis obj does not yield satisfactory results

### ISSUE THREAD

**Issue by sparshgarg23** [shunsukesaito/PIFu#138]
So,I tried training PiFU on the sample dennis obj,after training for 70 epochs this is my current result.Is this what you were getting for that particular object.
Compared to the actual dennis object,I feel that a lot of the features especially in the face,hand and front view section were missed.Plus the entire model looks sort of blocky.

Dennis reconstruction my result
![dennis_model_78epochs](https://user-images.githubusercontent.com/34626942/180744686-2d2002c1-dabe-4c59-b673-276fc55a6c5f.png)
The ground truth dennis model is as follows
![ground_truth_PIFU_front_view](https://user-images.githubusercontent.com/34626942/180744852-bf1fddba-8c22-40a9-830c-8b385a2ef929.png)
Note in the dennis folder I have only included one image which is 
rp_dennis_posed_004_A

**Comment by BhaumikThakkar** [comment]
hello there, can you tell me exact parameter which you shown in the fi0gure if you don't mind, because I am also training the model however it is taking too much time.

**Comment by sparshgarg23** [comment]
I didn't make any changes in the training config as such,however I changed the marching cube method to lewiner

**Comment by BhaumikThakkar** [comment]
oh I see, which device did you use like google colab or nvidia graphics laptop or any other?

**Comment by sparshgarg23** [comment]
colab

**Comment by BhaumikThakkar** [comment]
Great!! could you please help me out  if you have time? I am also working on colab however every time there was a memory issue and it crash the session?So I tried different platforms like paper space , Saturn cloud and other. So, can we connect please!  My email id is bhaumik112atgmaildotcom 
Thank You.

**Comment by sparshgarg23** [comment]
are you running on colab pro? I usually run my expriments on those platforms.Free version of colab won't help .Not sure about other platforms as I haven't used them.

**Comment by BhaumikThakkar** [comment]
Yeah I am running on free version, so you are using colab pro 
yeah!! okay no issue anyway thanks for you help!!

**Comment by sparshgarg23** [comment]
free version won't be of much use for heavier experiments.would recommend you invest in colab pro,it's only 11$(including tax)

**Comment by BhaumikThakkar** [comment]
> free version won't be of much use for heavier experiments.would recommend you invest in colab pro,it's only 11$(including tax)

Yeah actually I am thinking about that too. Will do it and let you know the results 😇✌️.

**Comment by BhaumikThakkar** [comment]
Hey @sparshgarg23, I tried in pro but not getting the output instead it took my 30 units 😅😅 out of 100 so can you please help me out, if you don't mind?!
